install_centos.md

```diff
@@ -0,0 +1,123 @@
+# Installing st2 on centos/rhel linux
+
+Follow this guide and we'll lead you through the basic installation and setup steps.
+
+## Setting up repos and fetching required software
+
+First let's update the system and fetch software which will be required during installation and setup process:
+```shell
+sudo yum makecache fast
+sudo yum install -y wget httpd-tools
+```
+
+Second, add repositories and install st2.
+```shell
+## NB! This step provides instructions for centos7 distro
+# Don't forget to use correct distro (el6 - for rhel6/centos6, and el7 - for rhel7/centos7)!
+export DISTRO=el7
+sudo wget https://bintray.com/stackstorm/${DISTRO}_staging/rpm -O /etc/yum.repos.d/bintray-stackstorm-${DISTRO}_staging.repo
+# Fix to use the correct URL
+sed -i "s|/${DISTRO}_staging|/${DISTRO}_staging/stable|" /etc/yum.repos.d/bintray-stackstorm-${DISTRO}_staging.repo
+unset DISTRO
+
+# Update repo and install st2
+sudo yum install -y st2
+```
+
+## Install dependent services
+
+St2 uses MongoDB database and RabbitMQ messaging queue, so we need to install these services. We suggest to use [IUS repo](https://ius.io/), it contains MongoDB and RabbitMQ. It's worth mentioning that IUS repo is *EPEL*-compatible, because EPEL comes as its part. So let's install the repo and proceed with the installation:
```

Comment:
Why custom repo?

This is good one for mongo:
https://github.com/enykeev/st2testground/blob/master/bootstrap-el6.sh#L9-L15
https://github.com/enykeev/st2testground/blob/master/bootstrap-el7.sh#L9-L15
and default for rabbitmq.